packages/x-components/src/x-modules/search/components/index.ts

```diff
@@ -7,8 +7,5 @@ export { default as Promoted } from './promoted.vue';
 export { default as PromotedsList } from './promoteds-list.vue';
 export { default as Redirection } from './redirection.vue';
 export { default as ResultsList } from './results-list.vue';
-export { default as SortDropdown } from './sort-dropdown.vue';
-export { default as SortList } from './sort-list.vue';
-export { default as SortPickerList } from './sort-picker-list.vue';
```

Comment:
We can't remove the components from here, otherwise would not be possible to import them when using the x-components build.

Reply:
That's right, restored